addons/l10n_latam_check/wizards/account_payment_register_views.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_payment_register_form" model="ir.ui.view">
+        <field name="name">account.payment.register.form</field>
+        <field name="model">account.payment.register</field>
+        <field name="inherit_id" ref="account.view_account_payment_register_form"/>
+        <field name="arch" type="xml">
+            <field name="payment_method_line_id" position="after">
+                <field name="l10n_latam_use_checkbooks" invisible="1"/>
+                <field name="payment_method_code" invisible="1"/>
+                <field name="l10n_latam_checkbook_type" invisible="1"/>
+                <field name="l10n_latam_checkbook_id" options="{'no_create': True, 'no_open': True}"
+                        attrs="{'invisible': ['|', ('payment_method_code', '!=', 'check_printing'), ('l10n_latam_use_checkbooks', '=', False)],
+                        'required': [('payment_method_code', '=', 'check_printing'), ('l10n_latam_use_checkbooks', '=', True)]}"
+                        domain="[('journal_id', '=', journal_id)]"/>
```

Comment:
Please try to use a consistent indentation
```suggestion
                <field name="l10n_latam_checkbook_id" options="{'no_create': True, 'no_open': True}"
                       attrs="{'invisible': ['|', ('payment_method_code', '!=', 'check_printing'), ('l10n_latam_use_checkbooks', '=', False)],
                               'required': [('payment_method_code', '=', 'check_printing'), ('l10n_latam_use_checkbooks', '=', True)]}"
                       domain="[('journal_id', '=', journal_id)]"/>
```

Reply:
should be done